### PR TITLE
Restore icons in the new filter popup

### PR DESCRIPTION
**Pull request by uladzimirdev**
### Description

I accidentally replaced icons with the only `label` icon in https://github.com/metabase/metabase/pull/42107
this PR restores correct behaviour

### How to verify

open dashboard -> open "add new filter" popup -> options should have correct icons

### Demo

image

version 0.49.5

image

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  },
+];
+
+describe("ParameterPopover", () => {
+  it("should render options with icons", () => {
+    render(
+      <ParametersPopover onClose={jest.fn()} onAddParameter={jest.fn()} />,
+    );
+
+    options.forEach(({ title, subtitle, icon }) => {
```
Comment (ranquild) [inline frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx:39]: it.each?

File: frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      expect(screen.getByText(title)).toBeInTheDocument();
+
+      const section = screen.getByText(title).parentElement as HTMLElement;
+
+      expect(within(section).getByText(subtitle)).toBeInTheDocument();
+      expect(
+        within(section).getByLabelText(`${icon} icon`),
```
Comment (ranquild) [inline frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx:46]: `getIcon` from `__suport__/ui`

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx (hunk 2 above)
Comment (uladzimirdev) [reply frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx:46]: I need to chain it with `within`

File: frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx (hunk 1 above)
Comment (uladzimirdev) [reply frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx]: [`1b57e07` (#44025)](https://github.com/metabase/metabase/pull/44025/commits/1b57e07ad64dca53f122ebc05f63b85391787f39)

**Review by mazameli — APPROVED** [review]

**Pull request merged**